server/bin/test-bin/cat

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if [[ ! -z "${1}" && "$(basename ${1})" == "tarball-bad-prefix-file_1970.01.01T00.00.00.prefix" ]]; then
+    exit 1
+fi
```

Comment:
I must have missed something, but the context and significance of this isn't obvious. Maybe a comment?